local/rest_api_gcbm/app.py

```diff
@@ -599,6 +599,41 @@ def config_table():
     except Exception:
         return {"status": 0, "error": Exception}
 
+@app.route("/upload/db/tables", methods=["GET"])
+def send_table():
+    """
+    Get GCBM table names from database file
+    ---
+    tags:
+            - gcbm
+    responses:
+            200:
+    parameters:
+            - in: Params
+            name: title
+                    type: string
+            description: GCBM table output 
+    """
+    # Default title = simulation
+    title = request.args.get("title")
+    input_dir = f"{os.getcwd()}/db/{title}"
```

Comment:
```
title = request.form.get("title") or "simulation"
input_dir = f"{os.getcwd()}/input/{title}/"
conn = sqlite3.connect(input_dir + "gcbm_input.db")
```

1. `title` is the name of the simulation, might be given by the user or randomly assigned, as in case of stress testing using locust
2. `gcbm_input.db` is the expected name of the database, it is present in `/input/run4/`